src/snowflake/connector/connection.py

```diff
@@ -475,8 +488,16 @@ def login_timeout(self) -> int | None:
         return int(self._login_timeout) if self._login_timeout is not None else None
 
     @property
-    def network_timeout(self) -> int | None:
-        return int(self._network_timeout) if self._network_timeout is not None else None
+    def request_timeout(self) -> int | None:
+        return int(self._request_timeout) if self._request_timeout is not None else None
```

Comment:
well, changing the property name is a BCR change.
honestly I don't think changing the name of a property is worth a major version bump
so we probably should just keep the `network_timeout`..
sorry I didn't realize you were about to to rename the property and should call it out sooner